src/Form/Destination/CommonITILField/ContentField.php

```diff
@@ -69,6 +69,7 @@ public function renderConfigForm(
                 options|merge({
                     'enable_richtext': true,
                     'enable_images': false,
+                    'enable_form_tags': true,
```

Comment:
For now, we can only configure the ticket's title and content, however we are planning to add a lot more configuration options.

For exemple, it could be possible to configure one or multiple tasks/followups that will be added to the ticket when it is created.

These fields would then use this new option.